spec/requests/api/v1/transition_checker/emails_spec.rb

```diff
@@ -83,7 +94,7 @@
       let(:user) { FactoryBot.create(:user, :confirmed) }
 
       context "with an email subscription" do
-        let(:subscription) { FactoryBot.create(:email_subscription, user_id: user.id) }
+        let!(:subscription) { FactoryBot.create(:email_subscription, user_id: user.id) }
```

Comment:
interesting, never used that with the exclaimation mark, what is this doing?

Reply:
It's essentially a combination of `let` and `before` - it runs the block before the tests, so the thing exists in the database when they start.  Normally `let` only runs the block when it's referenced.